dev-lang/yab/yab-1.7.8.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="Extended version of the yabasic programming language"
+DESCRIPTION="yab is an extended version of yabasic, a BASIC programming \
+language, with special commands designed for Haiku. "
+HOMEPAGE="http://software.besly.de/"
+COPYRIGHT="1995-2006 Marc-Oliver Ihm (yabasic)
+	2006-2009 Jan Bungeroth (yab improvements)
+	2013-2018 Jim Saxton (yab improvements)
+        2018-2020 Lorenz Glaser (yab improvements)"
```

Comment:
Use tab instead of spaces here (missed this one) :)
```diff
-         2018-2020 Lorenz Glaser (yab improvements)"
+	2018-2020 Lorenz Glaser (yab improvements)"
```